plaso/engine/artifact_filters.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe copy this to a separate PR and review this first?

This PR is getting big

Reply:
Done

Reply:
Merged in #1883